.studio/applications-service

```diff
@@ -161,61 +161,61 @@ function applications_populate_database() {
   for i in $(seq 1 12); do
     applications_publish_supervisor_message --name nginx --group "kansas-prod"\
       --origin custom --version "1.0.1" --release "20190115184823" --health 2\
-      --sup-id "$(uuidgen)"
+      --sup-id "$(uuidgen)" --site "testsite" --channel "stable"
```

Comment:
All these options are the same one for all our canned data inside this function. Can we make them a local function variable for easy editing later on?

Another idea is to not add these fields, site and channel, and instead put a default in the `applications-publisher` binary. (as I did for the `fqdn`)

Reply:
they actually do vary somewhat throughout the generation. Not by a lot though.  I wanted to have some be blank as well for ui testing. Which is why I didn't default.